src/inews-mixins/playlist.ts

```diff
@@ -152,15 +152,15 @@ export function GetRundownPlaylistInfoWithMixins(
 				expectedStart: lastRundownTiming.expectedStart
 			}
 		}
-		let result =
-			(getRundownPlaylistInfo ? getRundownPlaylistInfo(context, rundowns, '') : undefined) ??
-			literal<BlueprintResultRundownPlaylist>({
-				playlist: literal<IBlueprintResultRundownPlaylist>({
-					name: (rundowns[0] ?? { name: '' }).name,
-					timing
-				}),
-				order: null
-			})
+		let result: BlueprintResultRundownPlaylist = (getRundownPlaylistInfo
+			? getRundownPlaylistInfo(context, rundowns, '')
+			: undefined) ?? {
+			playlist: literal<IBlueprintResultRundownPlaylist>({
```

Comment:
Carefull with the nested ternaries and coalescing operators. This is starting to be messy to look at. 
Also, I think the indentation is slightly off here. It should be:
```
: undefined) ?? {
    playlist: {...}
```
not
```
: undefined) ?? {
playlist: {...}
```